api/routes/model.js

```diff
@@ -0,0 +1,193 @@
+const express = require("express");
+const router = express.Router();
+const validate = require("express-jsonschema").validate;
+
+const ModelSchema = {
+  id: "/ModelSchema",
+  type: "object",
+  properties: {
+    name: {
+      type: "string",
+      required: true
+    },
+    sdg: {
+      type: "array",
+      items: {
+        type: "integer"
+      },
+      required: true
+    },
+    description: {
+      type: "string",
+      required: true
+    },
+    stages: {
+      type: "object",
+      patternProperties: {
+        ".*": {
+          type: "object",
+          properties: {
+            stakeholders: {
+              type: "array",
+              items: {
+                type: "string"
+              }
+            },
+            challenges: {
+              type: "array",
+              items: {
+                type: "string"
+              }
+            },
+            insights: {
+              type: "array",
+              items: {
+                type: "string"
+              }
+            },
+            description: {
+              type: "string"
+            }
+          }
+        }
+      }
+    }
+  }
+};
+
+router.get("/", function (req, res, next) {
+  const db = req.db;
+  const collection = db.get("modelCollection");
+  collection.find({}, {}, function (e, docs) {
+    res.send(docs);
+  });
+});
+
+router.get("/:model_ID", function (req, res, next) {
+  const db = req.db;
+  let id = req.params.model_ID;
+  const collection = db.get("modelCollection");
+  collection.find({
+      _id: id
+    }, {
+      $exists: true
+    },
+    function (e, docs) {
+      if (docs) {
+        res.send(docs);
+      } else {
+        res.sendStatus(400);
+      }
+    }
+  );
+});
+//GET models by SDG
+router.get("/sdg/:sdg_num", function (req, res, next) {
```

Comment:
can we do this in the params of the first get request rather than a separate route?
using `?sdg=___`